Realm/RLMAccessor.mm

```diff
@@ -29,6 +29,23 @@
 
 #import <objc/runtime.h>
 
+typedef NS_ENUM(char, RLMAccessorCode) {
+    RLMAccessorCodeChar,
```

Comment:
We don't support char in objc - and I don't think we can since BOOL is represented as char.